implementation/tracing/src/connector_impl.cpp

```diff
@@ -280,9 +273,12 @@ void connector_impl::trace(const byte_t *_header, uint16_t _header_size,
                 for(int i = 0; i < its_data_size; i++) {
                     ss << ' ' << std::setfill('0') << std::setw(2) << std::hex << int(_data[i]);
                 }
-                std::string app = runtime::get_property("LogApplication");
-
-                ALOGI(app.c_str(), ss.str().c_str());
+                #ifdef ANDROID
+                    std::string app = runtime::get_property("LogApplication");
+                    ALOGI(app.c_str(), ss.str().c_str());
+                #else
```

Comment:
i however wanted you to try one change, and see if it works on your side:
```suggestion
```
if you remove this part, the VSOMEIP_TRACE should still print on ANDROID, since it has a ifdef guard for android logs

Reply:
Sorry, I'm not sure I follow. Do you mean we should remove that inner ifdef clause where we either log using ALOGI or the VSOMEIP_INFO/TRACE log macro? 

Instead of:

```
#ifdef ANDROID
  std::string app = runtime::get_property("LogApplication");
  ALOGI(app.c_str(), ss.str().c_str());
#else
  VSOMEIP_INFO << ss.str();
#endif
```

Just do:

```
VSOMEIP_INFO << ss.str();
```

BTW, I intentionally opted for using VSOMEIP_INFO here rather than VSOMEIP_TRACE to align with the current master that uses INFO level logging in the Android case (ALOGI).

Reply:
Exactly, if you look into the VSOMEIP_INFO, it already prints for Android:https://github.com/COVESA/vsomeip/blob/637fb6ccce969f89621660dd481badb29a90d661/implementation/logger/src/message.cpp#L144

does this work on your environment?

Reply:
Sure, that would work for us. 

However, I figured that the Android log macros were probably redefined in this file for a reason? Please see current master:  https://github.com/COVESA/vsomeip/blob/master/implementation/tracing/src/connector_impl.cpp#L28

I'm guessing the macros are redefined there to be able to add a specific log tag in this case instead of the default app name. Please see current master: https://github.com/COVESA/vsomeip/blob/master/implementation/tracing/src/connector_impl.cpp#L283

That being said, I don't see the immediate benefit of using a different log tag. So if you prefer, in the interest of keeping this commit as small as possible, I can create another pull request to remove the redefinition AND use of the Android log macro in this file?

Reply:
@falk-haleytek isn't that what also is done here? https://github.com/COVESA/vsomeip/blob/637fb6ccce969f89621660dd481badb29a90d661/implementation/logger/src/message.cpp#L145

Reply:
Sorry, you are completely right! I don't know how I misread that and thought they used different log tags for some reason :) 

I'll update the PR to only do VSOMEIP_INFO << ss.str(); then. 

I guess I should also remove the redefinition of ALOGI in this file then or did I miss the reason for them being there?

Reply:
yes you can remove it!

Reply:
Done! https://github.com/COVESA/vsomeip/pull/726/commits/f37185a2b799e029b0de428aeaad34c2e86a2941